README.md

```diff
@@ -20,10 +20,20 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   cd ..
   catkin_make
   ```
+  
+### Application can work at two modes:
+-  Local mode
+  -  All the ROS nodes run at the same machine.
+-  Network mode
+  -  ROS nodes run at two different machine: Server and Client.
+  -  Client runs ROS nodes "RVIZ" and "point_cloud_generator_node".
+  -  Server runs all other ROS nodes except two nodes running on client.
```

Comment:
I think the problem might be in the move_group node which looks for the** join_trajectory_action** topic on startup.  Then perhaps you could run this on the server machine only after the client has been started.
In regards to separating the client into two parts, I don't see how that would help solve this initialization issue that you are having. I do think that it is a realistic scenario but for now let's focus on getting things to work towards the modes that we've outlined  In order to keep things organized.  We can address separating the client in a subsequent pull request after we complete this one.